lib/commands/log.js

```diff
@@ -41,7 +41,7 @@ class LogCommand extends Command {
             }
 
             const logFileName = path.join(instance.dir, 'content/logs', `${instance.config.get('url').replace(/[^\w]/gi, '_')}_${this.system.environment}${argv.error ? '.error' : ''}.log`);
-            const prettyStream = new PrettyStream();
+            const prettyStream = new PrettyStream({color: argv.color});
```

Comment:
don't think this line needs to be here? Since prettystream now uses chalk under the hood it should pick things up automatically.

Reply:
Oops 🙈 fixed!